frontend/src/lib-components/atoms/form/Checkbox.tsx

```diff
@@ -18,7 +18,7 @@ const Wrapper = styled.div`
   cursor: pointer;
 
   label {
-    font-size: 15px;
+    font-size: 16px;
```

Comment:
I think using `1rem` is a better idea here, because the reason this didn't match the spec is that font size was increased from 15px to 16px at one point.

Reply:
Good point! Fixed.